### PR TITLE
add onnx simplify option to pytorch2onnx tool

### PR DESCRIPTION
**Pull request by RunningLeon**
This PR simply add `simplify` option to pytorch2onnx tool. Users could use this feature to simplify the exported onnx models.

### CODE REVIEW

File: docs/tutorials/pytorch2onnx.md [hunk 1]
```diff
@@ -90,6 +91,7 @@ Notes:
 
 ## Reminders
 
+- `mmcv.onnx.simplify` feature is based on [onnx-simplifier](https://github.com/daquexian/onnx-simplifier). If you want to try it, please refer to [onnx in `mmcv`](https://github.com/open-mmlab/mmcv/blob/master/docs/onnx.md) and [onnxruntime op in `mmcv`](https://github.com/open-mmlab/mmcv/blob/master/docs/onnxruntime_op.md) for more information.
```
Comment (ZwwWayne) [inline docs/tutorials/pytorch2onnx.md:94]: May use the documentation in readthedocs.

File: tools/pytorch2onnx.py [hunk 2]
```diff
@@ -20,7 +20,8 @@ def pytorch2onnx(config_path,
                  verify=False,
                  normalize_cfg=None,
                  dataset='coco',
-                 test_img=None):
+                 test_img=None,
+                 do_simplify=False):
```
Comment (ZwwWayne) [inline tools/pytorch2onnx.py:24]: do_simplify -> simplify

File: tools/pytorch2onnx.py [hunk 3]
```diff
@@ -52,10 +53,31 @@ def pytorch2onnx(config_path,
         opset_version=opset_version)
 
     model.forward = orig_model.forward
+
+    # simplify onnx model
+    if do_simplify:
+        from packaging import version
```
Comment (ZwwWayne) [inline tools/pytorch2onnx.py:59]: Seems to be new dependency. We may try not to use it by following [here](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/__init__.py).

File: tools/pytorch2onnx.py [hunk 4]
```diff
@@ -52,10 +53,31 @@ def pytorch2onnx(config_path,
...
-        from mmdet.core import get_classes
+        from mmdet.core import get_classes, bbox2result
         from mmdet.apis import show_result_pyplot
+
+        ort_custom_op_path = ''
+        try:
+            from mmcv.ops import get_onnxruntime_op_path
+            ort_custom_op_path = get_onnxruntime_op_path()
+        except (ImportError, ModuleNotFoundError):
+            pass
```
Comment (ZwwWayne) [inline tools/pytorch2onnx.py:79]: If we pass here, what would happen? Should we raise an warning?

Comment (RunningLeon) [reply tools/pytorch2onnx.py]: Added a warning. It's Ok to pass if the input model does not have a custom op from `mmcv`.

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**